frontend/src/features/feature-flags/queries.ts

```diff
@@ -11,3 +11,16 @@ export const featureFlagsKeys = {
 export const useFeatureFlags = (): UseQueryResult<Set<string>> => {
   return useQuery(featureFlagsKeys.base, () => getEnabledFeatureFlags())
 }
+
+// Retrieves a feature flag and returns the default value
+// if an error is encountered or if the flag does not exist
+export const useFeatureFlagWithDefaults = (
```

Comment:
With a more generic name, I think we should use this method for the rest of the code. Every flag should have an explicitly defined default.

```
// definition
useIsFeatureEnabled(flagName: string,  defaultValue: boolean): boolean

// usage
const enableTurnstile = useIsFeatureEnabled(
    featureFlags.turnstile,
    false,
  )
```

Reply:
Sure, I can add the definition above. I was also thinking of replacing the existing feature flags calls with the new function. I can create another ticket for this and do it as part of a separate diff. What do you think?

Reply:
I would prefer that to be done in a separate ticket -- just to avoid feature creep 😆